### ISSUE TITLE
Favorite attribute not synced between app and server

### ISSUE THREAD

**Issue by mgerhardy** [nextcloud/ios#48]
It would be nice to also get the favorite flags synchronized from the app to nextcloud and vice versa.

**Comment by mgerhardy** [comment]
#46  might be related.

**Comment by marinofaggiana** [comment]
hi @mgerhardy  favorite (sync server) is in to.do list ;)

**Comment by jancborchardt** [comment]
Yep, this is very crucial for consistency between server and apps.

**Comment by mr-manuel** [comment]
I have the behavior, that the favorites (not added in the app but on the server) are appearing in the app only after I browsed them under "Files". Is this the same thing?

**Comment by marinofaggiana** [comment]
@mr-manuel I don't understand ...

**Comment by mr-manuel** [comment]
When I clear the cache and then open the "Favorites" folder in the app the list is empty, but with the latest update 2.17 (23) it works :)

**Comment by jancborchardt** [comment]
This issue is about:
- When you mark a file on the server as favorite, it should also automatically be marked as favorite in the iOS app
- Vice versa too, if you mark a file in the iOS app as favorite, it should automatically be marked as favorite on the server too.

And on the iOS app, being marked as favorite should mean »keep this file synced and available offline always«.

**Comment by marinofaggiana** [comment]
@jancborchardt it's already so now

**Comment by jancborchardt** [comment]
@marinofaggiana so this issue is fixed?

@mgerhardy @mr-manuel can you confirm it works for you with the latest update? :)

**Comment by mr-manuel** [comment]
I installed the latest version and cleared the cache. Now it works as it should :)

**Comment by jancborchardt** [comment]
@mr-manuel thanks for the confirmation! :) Can you open a new issue for that crash, ideally with more info? :) (iOS version, app version etc, if you’re part of the beta, and so on)

**Comment by marinofaggiana** [comment]
@mr-manuel @jancborchardt 

https://github.com/nextcloud/ios/issues/143
